index.mako

```diff
@@ -1633,6 +1637,28 @@ immutable caches:
         key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
 ```
 
+### circleci example
+
+like [azure pipelines](#azure-pipelines-example), circleci also uses
+immutable caches:
+
+```yaml
+steps:
+    - run:
+        command: |
```

Comment:
this indentation doesn't line up properly (either use 2 or 4 space indents, but not mixed)

Reply:
Fixed.